src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -557,6 +558,22 @@ export const noFDCWarning = (
   </div>
 );
 
+
+const options = [
+  { id: 1, label: 'first' },
+  { id: 2, label: 'second' },
+  { id: 3, label: 'third' },
+  { id: 4, label: 'fourth' },
```

Comment:
Do we have any guidance regarding how many of these can be added?

Reply:
I'm not sure I understand the question. These are just dummy values returned by our fake api call. I was using these for testing; we'll get rid of them when we hook it up to the api for real.

I'm not sure how many options the autosuggest field can handle, if that's your question. I think I saw an option somewhere to limit the suggestions shown.

Reply:
Thanks for explaining.